### DISCUSSION TITLE
[Feature Request] Database Import Feature

### DISCUSSION THREAD

**Discussion by Dvalin21** [tobychui/zoraxy#184] [Ideas]
Not sure how difficult or easy this would be, but a import feature to import reverse proxy list from other reverse proxies to this one, when switching from those to this one. I find myself in this situation. Thanks.

**Comment by tobychui** [comment]
That is really unlikely to happen, as you will need a tons of converter scripts to match the number of reverse proxy solutions out there * the number of version they have * the number of version Zoraxy have. Even if you only implement the latest version conversion (e.g. latest version of NPM to latest version of Zoraxy), you will still need to update the script every time anyone has a new release. 

The best way for you to do the migration is to upload the config of your previous reverse proxy and an example of Zoraxy proxy config json file to a GPT or LLM model and let them figure out it for you.
